Python/Product/Common/Strings.resx

```diff
@@ -2935,4 +2935,7 @@ Please specify at least one package.</value>
     <value>A file containing Python tests was detected in the {0} {1}, but the test framework is disabled.</value>
     <comment>{0} is project or workspace name. {1} is "project" or "workspace"</comment>
   </data>
+  <data name="PythonDiscoveryResultsNotFound" xml:space="preserve">
+    <value>Discovery results file '{0}' was not found {</value>
```

Comment:
there's a extra curly brace at the end (and missing period)

Reply:
should i add a period too?

Reply:
yeah